### ISSUE TITLE
Error when analyzing using tensorflow 2.4 

### ISSUE THREAD

**Issue by aliyaHammad** [albermax/innvestigate#242]
Hello! 

I have been trying to apply an analyzer on a model using TensorFlow 2.4 and I am getting the error shown in the attached pictures. I am using the innvestigate_updates_towards_tf2.0 branch and it works when I am using TensorFlow 2.2, but when upgrading to 2.4 it seems to crash. I can only load the model using TensorFlow 2.4 so it is not possible to just downgrade to 2.2. 

Creating an analyzer works fine, but it's the analyzer.analyze(image) function that is causing the issue. 

Thanks! 

![Bildschirmfoto von 2021-03-12 23-18-22](https://user-images.githubusercontent.com/21117989/111005160-97c61a00-838a-11eb-974f-add1c3ba49f4.png)
![Bildschirmfoto von 2021-03-12 23-18-34](https://user-images.githubusercontent.com/21117989/111005181-9f85be80-838a-11eb-93d6-60dfa4556d59.png)

**Comment by leanderweber** [comment]
Hi @aliyaHammad,

the innvestigate_updates_towards_tf2.0  branch was implemented for (up to) tensorflow version 2.3, so there my be some issues when using tensorflow 2.4. The branch will not really be updated anymore going forward, since an official innvestigate 2 version is currently being worked on.

However, even with tensorflow 2.4, I was not able to reproduce above bug, could you maybe provide some more information?

Cheers,
Leander

**Comment by aliyaHammad** [comment]
Hi @leanderweber! 

Thank you for answering. I was using an older version of the branch, and once updating it I faced another issue.

![Bildschirmfoto von 2021-04-20 22-11-10](https://user-images.githubusercontent.com/21117989/115459156-036f9280-a227-11eb-9d77-0caa1c3844b9.png)

I also noticed that the innvestigate.utils.keras.graph.model_wo_softmax() function deos not actually strip the model of the softmax layer. This issue never happened in the last version. I am using Pycharm, tensorflow 2.4 and python 3.8. 

Regards,
Aliya Hammad
